### ISSUE TITLE
Build: Lock and unlock both have same symbols

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#77]
URL: https://hcdev.openplus.ca/fr/node/2525/build#Section1 (click on Build for any Recall or Alert from Dashboard)
User Agent: Chrome, Firefox
Language: Both

Issue Description: Lock and edit & Unlock both have the same graphic with the lock that is locked. This is confusing for the visual user. 

![image](https://user-images.githubusercontent.com/14280062/129226019-46867e75-5993-40a1-b73a-383f44d630ff.png)

![image](https://user-images.githubusercontent.com/14280062/129226056-6e0e7eeb-f914-40b8-ab42-7ec6a58e0325.png)

Expected results: One of these should have the lock open (unlocked).

**Comment by echouin** [comment]
Fixed in final version.

![image](https://user-images.githubusercontent.com/88099454/148240090-3dc920e6-1d40-4449-942e-2d3cef7e1df0.png)

![image](https://user-images.githubusercontent.com/88099454/148240111-851472f1-b4d9-4c07-a41a-a0d9ed98ff2d.png)

**Comment by echouin** [comment]
@mmiron-HC here's an easy one to close :)

**Comment by davidgotlieb** [comment]
@echouin was happy you flagged this one, he found it irritating too. :)

**Comment by ndolter** [comment]
Validated that the unlocked button graphic has been corrected in both French and English.
